donut/templates/campus_positions.html

```diff
@@ -93,3 +135,23 @@ <h1 id="pagetitle" > The Ascit Position Service </h1>
     {{all_positions|safe}});
 </script>
 {% endblock %}
+{% block scripts %}
+    {{ super()  }}
+    <script src='{{ url_for("static", filename="js/search-directory.js") }}'></script>
+    <script>
+        $(document).ready(function() {
+            var nameInput = $('input#name'), nameSearch = $('ul#name-search')
+            attachDirectorySearch(nameInput, nameSearch, function(user){
+                return $('<li>').addClass('list-group-item').append($('<a>')
+                        .text(user.full_name + ", " + user.graduation_year)
+                        .click({name: user.full_name, user_id: user.user_id},
```

Comment:
I don't think you need to bind the user fields here; just use user in the callback instead of event.data